### PR TITLE
Feature/add sidebar

### PR DESCRIPTION
**Pull request by JamesDMayo**
Add a sidebar and prep for Wiki Api

### CODE REVIEW

File: ui/src/App.js [hunk 1]
```diff
@@ -1,25 +1,58 @@
...
+import cloudData from './Word-Cloud/cloudData';
+import cloudController from './controllers/cloudController';
+
 
 function App() {
+
+  const [mainWord, setMainWord] = useState(cloudData.start);
+  const [cloudWords, setCloudWords] = useState(cloudData.data);
+  const [currentWord, setCurrentWord] = useState([cloudData.start]);
+  // const [wikiWords, setWikiWords] = useState();
```
Comment (Zachariasp) [inline ui/src/App.js:16]: Remove unused code

File: ui/src/App.js [hunk 2]
```diff
@@ -1,25 +1,58 @@
...
+
+
   
+  const handleClick = (tagValue) => {
+    // e.preventDefault();
```
Comment (Zachariasp) [inline ui/src/App.js:21]: Remove unused code

File: ui/src/components/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// import { cloudStart, cloudFinish } from "../Word-Cloud/cloudData";
```
Comment (Zachariasp) [inline ui/src/components/Sidebar.jsx:1]: delete

**Review by Zachariasp — CHANGES_REQUESTED** [review]
See comments. Otherwise, Great WOrk kiddo!

File: ui/src/components/Sidebar.jsx (hunk 3 above)
Comment (JamesDMayo) [reply ui/src/components/Sidebar.jsx:1]: I handled it.

File: ui/src/App.js (hunk 2 above)
Comment (JamesDMayo) [reply ui/src/App.js:21]: I handled it.

File: ui/src/App.js (hunk 1 above)
Comment (JamesDMayo) [reply ui/src/App.js:16]: I handled it.

**Review by Zachariasp — APPROVED** [review]

**Pull request merged**